doc/dds-core-xtypes/README.md

```diff
@@ -120,94 +122,93 @@ The type consistency, could be a set of the following *QoS polities*:
 - `IGNORE_MEMBER_NAMES`: the evaluation will be true if the names of some members differs (but no the position).
 - `IGNORE_OTHER_MEMBERS`: the evaluation will be true if some members of `other_type` are ignored.
 
-Note: the `TypeConsistency` is an enum with `|` and `&` operator override to manage it as a set of QoS polities.
+Note: `TypeConsistency` is an enum with `|` and `&` operators overrided to manage it as a set of QoS polities.
 
-### Data instantation
+### Data instances
 #### Initialization
 To instantiate a data, only is necessary a `DynamicType`:
 ```c++
 DynamicData data(my_defined_type);
 ```
 
-This line allocates all the memory necesary to hold the data of `my_defined_type`
-and initialize their content to *0* or to the corresponding *default values*.
-It is important to know that the type must have a higher lifetime than the DynamicData,
-because the DynamicData only save a references to it.
-Other ways to initalizate a `DynamicData` is by *copy*, and by *compatible copy*.
+This line allocates all the memory necesary to hold the data defined by `my_defined_type`
+and initializes their content to *0* or to the corresponding *default values*.
+Please, note that the type must have a higher lifetime than the DynamicData,
+since `DynamicData` only saves a reference to it.
+Other ways to initalizate a `DynamicData` are respectively *copy* and *compatible copy*.
 
 ```c++
 DynamicData data1(type1); //default initalization
 DynamicData data2(data1); //copy initialization
 DynamicData data3(data1, type2); //compatible copy initialization
 ```
 
-Last line creates a compatible `DynamicData` with the values of `data1` that can be accessed as `type2`.
+The last line creates a compatible `DynamicData(type2)` with the values of `data1` that can be accessed being a `type2`.
```

Comment:
```suggestion
The last line creates a compatible `DynamicData` with the values of `data1` that can be accessed being a `type2`.
```
I would remove this part because could be confusion